static/app/views/organizationGroupDetails/groupEvents.spec.jsx

```diff
@@ -181,6 +181,32 @@ describe('groupEvents', function () {
       {context: routerContext, organization}
     );
     const perfEventsColumn = screen.getByText('transaction');
+    expect(discoverRequest).toHaveBeenCalledWith(
+      '/organizations/org-slug/events/',
+      expect.objectContaining({
+        query: expect.objectContaining({query: 'performance.issue_ids:1 '}),
+      })
+    );
     expect(perfEventsColumn).toBeInTheDocument();
   });
+
+  it('renders new events table if error', function () {
+    const org = initializeOrg();
+    org.features = ['performance-issues-all-events-tab'];
```

Comment:
```suggestion
    const org = initializeOrg({
      organization: {features: ['performance-issues-all-events-tab']}
    });
```

Reply:
@Kevan, hmm, this causes
<img width="710" alt="image" src="https://user-images.githubusercontent.com/44422760/193091966-0ee42e6c-6418-4a0d-a6e3-c1789539b949.png">

have to add this to initializeOrg to work
<img width="415" alt="image" src="https://user-images.githubusercontent.com/44422760/193092437-9893972b-c897-45c9-951e-5113cc0492a6.png">

Reply:
hm but we pass features like that in a bunch of places. why does it not work here 🤔

Reply:
figured it out, you have to do org.organization
<img width="527" alt="image" src="https://user-images.githubusercontent.com/44422760/193100568-8d6efb0b-5495-46cb-9da1-1186a08e5459.png">

Reply:
Kinda confusing, because the function is called initializeOrg so you'd expect it to return the organization, but it return the organization as a property.

Reply:
Regarless, fixed the issue!

Reply:
also, `org.org` is the same as `org.organization`??
<img width="207" alt="image" src="https://user-images.githubusercontent.com/44422760/193101062-e6fe648e-8f94-4aab-a55f-abafb1320f40.png">

Reply:
oh yeah that's odd. we usually read `organization` not `org`